src/bindings/python/src/pyopenvino/graph/model.cpp

```diff
@@ -733,6 +733,22 @@ void regclass_graph_Model(py::module m) {
                     :return: Index for value referencing it.
                     :rtype: int
                  )");
+    model.def(
+        "get_result_index",
+        [](const ov::Model& model, const ov::op::v0::Result& result) {
+            return model.get_result_index(result.get_default_output());
+        },
+        py::arg("result"),
+        R"(
+                Return index of result.
+
+                Return -1 if `result` not matched.
+
+                :param result: Result operation
+                :type result: openvino.runtime.op.Result
```

Comment:
```suggestion
                :type result: op.Result
```

Just to be consistent with other docstrings